spec/models/abuse_report_spec.rb

```diff
@@ -359,6 +359,13 @@
         .not_to have_enqueued_job
     end
 
+    it "does not attach a download for comment sub-URLs asynchronously" do
+      allow(subject).to receive(:url).and_return("http://archiveofourown.org/works/#{work.id}/comments/")
```

Comment:
Thanks! I notice on the Jira there's no trailing slash, and I think (assuming I read the regex right) that case would still fail. Would you mind updating this test case, or perhaps adding another that matches what's described in the linked Jira ticket?

Reply:
When sent through the web form, URLs have the trailing slash added in `abuse_report.rb#clean_url`, so since this test skips that step this should be accurate? I've tested the full form with all the examples in the ticket and it works as expected.

Reply:
You're right, thanks for pointing that out!